api/v2.0/swagger.yaml

```diff
@@ -110,6 +110,12 @@ paths:
           description: Query the artifacts by type. Valid values can be "IMAGE", "CHART", etc.
           type: string
           required: false
+        - name: tags
```

Comment:
Is it better to treat tags/untags as sub-resources of artifacts? Provide new API endpoints to handle such request?